index.html

```diff
@@ -279,11 +281,15 @@ <h2 id="h-tag" class="desktop-multipost">Uber Navigation</h2>
             <li class="small-langli">javascript</li>
           </ul>
           <div>
-            <button type="button" class="seeproject desktop-multipost active">See Project</button>
+            <button id="desktop4" onclick="displayModal(3)" type="button" class="seeproject desktop-multipost active">See Project</button>
           </div>
         </div>
       </div>
     </section>
+    <section>
```

Comment:
- [ Required ] According to the project requirements the `works` section should be created dynamically with the information from the projects array, currently, the works section in your project is `hardcoded` which should not be the case. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-03-10 11-45-29](https://user-images.githubusercontent.com/16392046/224268096-e68db726-3a00-4f36-a733-7abe005047f6.png)